### ISSUE TITLE
transformer_seq2seq get accuracy:0, is it normal?

### ISSUE THREAD

**Issue by yifang828** [arkilpatel/SVAMP#9]
2022-03-04 22:41:25,129 | INFO | logger.py: 39 : print_log() ::  
 Epoch: 50
2022-03-04 22:42:48,423 | DEBUG | model.py: 410 : train_model() ::       Training for epoch 50 completed...
Time Taken: 1.388223695755005
2022-03-04 22:42:48,423 | DEBUG | model.py: 411 : train_model() ::       Starting Validation
2022-03-04 22:42:58,624 | INFO | logger.py: 39 : print_log() ::  
 Epoch: 50
 best epoch: 0
 train loss epoch: 1.3217621725835618
 min train loss: 1.3217621725835618
 val loss epoch: 3.8505187034606934
 min val loss: 2.108802556991577
 train acc epoch: 0.0
 max train acc: 0.0
 val acc epoch: 0.0
 max val acc: 0.0
 val bleu epoch: (0.0, [0.07439365671641791, 0.009732360097323601, 0.0, 0.0], 1.0, 1.2350230414746544, 4288, 3472)
 max val bleu: 0.0
2022-03-04 22:42:58,624 | INFO | model.py: 489 : train_model() ::        Training Completed for 50 epochs

--Return--
> /home/wirl/patti/NTUT_workspace/paper_ref/SVAMP/code/transformer_seq2seq/src/utils/logger.py(86)store_results()->None
-> pdb.set_trace()
(Pdb) c
2022-03-07 13:51:08,462 | INFO | model.py: 493 : train_model() ::        Scores saved at ./out/val_results_mawps-asdiv-a_svamp.json

my execute command 
`python3 -m src.main -mode train -gpu 0 -embedding roberta -emb_name roberta-base -lr 0.0002 -emb_lr 8e-6 -batch_size 4 -epochs 50 -dataset mawps-asdiv-a_svamp -no-full_cv -run_name run_cv_asdiv-a`

i got every accuracy zero when i executed src.main under transformer_seq2seq folder with dataset mawps-asdiv-a_svamp
